crates/gitql-engine/src/engine_function.rs

```diff
@@ -315,21 +342,24 @@ fn select_diffs(
 }
 
 fn select_branches(
-    repo: &git2::Repository,
+    repo: &gix::Repository,
     fields_names: &Vec<String>,
     fields_values: &Vec<Box<dyn Expression>>,
     alias_table: &HashMap<String, String>,
 ) -> Vec<GQLObject> {
     let mut branches: Vec<GQLObject> = Vec::new();
-    let local_branches = repo.branches(None).unwrap();
+    let platform = repo.references().unwrap();
+    let local_branches = platform.local_branches().unwrap();
```

Comment:
We need function to get all branches (Local and Remote) to iterate on them

Reply:
There is [a separate method](https://docs.rs/gix/0.53.1/gix/reference/iter/struct.Platform.html#method.remote_branches) to iterate remote branches. To to their layout on disk it's fine to do that in two steps. Otherwise, [iterate all of them](https://docs.rs/gix/0.53.1/gix/reference/iter/struct.Platform.html#method.all) and [categorize later](https://docs.rs/gix-ref/0.36.0/gix_ref/struct.FullName.html#method.category_and_short_name).